qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -70,6 +71,13 @@ def __init__(self, parent: VisaInstrument, name: str, UID: str) -> None:
         super().__init__(parent, name)
         self.uid = UID
 
+        # The firmware update from 2.5 -> 2.6 changed the command
+        # syntax slightly
+        if LooseVersion(self.root_instrument.firmware) > LooseVersion('2.5'):
```

Comment:
I think this needs to be
```suggestion
        if LooseVersion(self.root_instrument.firmware) >= LooseVersion('2.6'):
```

Since 
```
 LooseVersion('2.5.08.000') > LooseVersion('2.5') 
```
Evaluates to True